OpenRA.Mods.Cnc/FileFormats/BlowfishKeyProvider.cs

```diff
@@ -1,6 +1,6 @@
 #region Copyright & License Information
 /*
- * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * Copyright 2009-2020 The OpenRA Developers (see AUTHORS)
```

Comment:
Last time this was modified was the copyright year header change. https://github.com/OpenRA/OpenRA/commit/23b3c237b7071fd308c4664b0b6c5d719c0f3c74#diff-6ead9da50ccf0af6b203a7393c0b3c41a29ae2c82f338934380eb3d9257c8169 That should be excluded as it is a non-copyrightable change.